test/grid_test_utils.h

```diff
@@ -0,0 +1,22 @@
+#include "grid.h"
+#include "test_utils.h"
+
+namespace celerity::test_utils {
+
+struct partition_vector_order {
```

Comment:
Unused? 🤔

Reply:
It is! I had attempted a "connected partition" optimization step that ended up on the cutting room floor. Removed.